test-integration/helpers/ui/test-element.coffee

```diff
@@ -28,14 +32,28 @@ class TestItemHelper
     else
       @locator
 
-  get: (args...) =>
+  aliasWait: (waitFunction, args...) =>
     locator = @getLocator(args...)
-    @test.utils.wait.for(locator)
+
+    waitTime = _.last(args)
```

Comment:
I'm a bit worried about having too many places to provide timeout data. Why not just use `giveTime` if needed?